taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/tasklistener/TaskanaTaskListener.java

```diff
@@ -162,7 +162,7 @@ private void setOutboxSchema(Connection connection) throws SQLException {
 
     String outboxSchemaName =
         ReadPropertiesHelper
-            .getSchemaFromProperties("taskana-outbox.properties",
+            .getSchemaFromProperties("taskana-task-listener.properties",
```

Comment:
I would prefer the name to be more generic, something like taskana-listener-properties or even camunda-listener-properties, since the TaskanaTaskListener as well as the ParseListenerProcessEnginePlugin both use this properties file, not just the tasklistener